Sources/AltSwiftUI/Source/Views/Controls/Menu.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Menu.swift
+//  AltSwiftUI
+//
+//  Created by Chan, Chengwei on 2021/02/25.
+//  Copyright © 2021 Rakuten Travel. All rights reserved.
+//
+
+import UIKit
+
+@available(iOS 14.0, *)
+/// A view that can be tapped by the user to open a menu.
+public struct Menu: View {
+    public var viewStore = ViewValues()
+    var labels: [View]
+    let viewContent: [View]
+    
+    /// Creates an instance that triggers an `action`.
+    ///
+    /// - Parameters:
+    ///     - - content: A view builder that creates the content of menu options.
+    ///     - label: The visual representation of the menu button
+    public init(@ViewBuilder content: () -> View, @ViewBuilder label: () -> View) {
+        self.labels = label().subViews
+        self.viewContent = content().subViews
+    }
+    
+    public var body: View {
+        self
+    }
+}
+
+@available(iOS 14.0, *)
+extension Menu {
+    /// Creates an instance with a `Text` visual representation.
+    ///
+    /// - Parameters:
+    ///     - title: The title of the button.
+    ///     - content: A view builder that creates the content of menu options.
+    public init(_ title: String, @ViewBuilder content: () -> View) {
+        labels = [Text(title)]
+        self.viewContent = content().subViews
+    }
+}
+
+@available(iOS 14.0, *)
+extension Menu: Renderable {
+    public func updateView(_ view: UIView, context: Context) {
+        guard let view = view as? SwiftUIMenuButton,
```

Comment:
You can reuse Button here and prevent having repeated button logic.

let button = Button(...)
button.updateView(...)
// Update uiButton menu <-- You are missing updating the menu, in case the menu changes